src/ui_parts/viewport.gd

```diff
@@ -51,6 +52,22 @@ func center_frame() -> void:
 	adjust_view()
 	set_view((SVG.root_tag.get_size() - size / Indications.zoom) / 2)
 
+func create_object_context() -> Popup:
+	var btn_array: Array[Button] = []
+	var new_tid := PackedInt32Array([SVG.root_tag.get_child_count()])
+	var mouse_position := (get_mouse_position() / Indications.zoom + view.position).snapped(Vector2.ONE * GlobalSettings.save_data.snap)
+	
+	for tag in [TagPath.new(), TagCircle.new(), TagEllipse.new(), TagRect.new(), TagLine.new()]:
+		if tag is TagPath:
+			tag.attributes["d"]._commands.append(PathCommand.MoveCommand.new(0.0, 0.0, false))
+		
+		tag.attributes["transform"].set_transform_list([AttributeTransform.TransformTranslate.new(mouse_position.x, mouse_position.y)] as Array[AttributeTransform.Transform])
+		btn_array.append(Utils.create_btn(tr(tag.name), SVG.root_tag.add_tag.bind(tag, new_tid), false, tag.icon))
```

Comment:
We don't need to use tr(tag.name) as tag names aren't translated right now, they are meant to be like the tags in code at the moment.